aws/resource_aws_lb_listener_rule.go

```diff
@@ -283,18 +284,157 @@ func resourceAwsLbbListenerRule() *schema.Resource {
 			"condition": {
 				Type:     schema.TypeSet,
 				Required: true,
+				Set:      lbListenerRuleConditionSetHash,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"field": {
-							Type:         schema.TypeString,
-							Optional:     true,
-							ValidateFunc: validation.StringLenBetween(0, 64),
+							Type:     schema.TypeString,
+							Required: true,
```

Comment:
Can we add a test to cover the `field` field being required, thus throwing an error?
As in an acceptance test with `ExceptError` set.

Ex: https://github.com/terraform-providers/terraform-provider-aws/blob/a31e76aa3c32e551459303cffa5ecbf90f7d7572/aws/resource_aws_cloudfront_distribution_test.go#L297. Let me know if you need more insights on that!

Reply:
Done in `TestAccAWSLBListenerRule_conditionNoField`